.github/workflows/ci.yaml

```diff
@@ -444,3 +440,60 @@ jobs:
         asset_name: tanzu-apps-plugin${{ matrix.os-arch }}-${{ env.RELEASE_VERSION }}.tar.gz
         asset_content_type: application/gzip
 
+  publish-oci-artifact:
+    env:
+      TKG_CUSTOM_IMAGE_REPOSITORY: projects-stg.registry.vmware.com/tkg
+      TANZU_CLI_NO_INIT: true
+      TANZU_HOME: $HOME/tanzu
+      OS_ARCH: linux-amd64
+    needs:
+    - release
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: vmware-tanzu/carvel-setup-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Install tanzu cli
+      run: |
+        TANZU_VERSION=$(cat TANZU_VERSION)
+        mkdir -p ${TANZU_HOME}
+        curl -Lo tanzu-framework.tar.gz https://github.com/vmware-tanzu/tanzu-framework/releases/download/${TANZU_VERSION}/tanzu-framework-linux-amd64.tar.gz
+          tar -xzf tanzu-framework.tar.gz -C ${TANZU_HOME}
+          sudo mv ${TANZU_HOME}/cli/core/${TANZU_VERSION}/tanzu-core-linux_amd64 /usr/local/bin/tanzu
+          chmod +x /usr/local/bin/tanzu
+        env
+        tanzu init
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set repos
+      run: |
+        echo "DISTRIBUTION_REPO=ghcr.io/${{ github.repository }}/distribution" >> $GITHUB_ENV
+        echo "DISCOVERY_REPO=ghcr.io/${{ github.repository }}/discovery" >> $GITHUB_ENV
+        echo $DISTRIBUTION_REPO
+        echo $DISCOVERY_REPO
+    - name: Build Version
+      run: echo "BUILD_VERSION=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+    - name: Download plugin bundle
+      uses: actions/download-artifact@v3
+      with:
+        name: tanzu-apps-plugin.tar.gz
+    - name: Install apps plugin
+      run: |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        set -u
+
+        echo "ARTIFACTS_DIR=./${OS_ARCH}" >> $GITHUB_ENV
```

Comment:
we do not need to set the variable to githubenv

Reply:
removed variable